wegodang/users/models.py

```diff
@@ -0,0 +1,23 @@
+from django.db import models
+
+class User(models.Model):
+    username     = models.CharField(max_length=45, unique=True)
+    first_name   = models.CharField(max_length=45)
+    last_name    = models.CharField(max_length=45)
+    email        = models.EmailField(max_length=200, unique=True)
+    password     = models.CharField(max_length=200)
+    phone_number = models.CharField(max_length=100, unique=True)
+    kakao_id     = models.BigIntegerField()
+    created_at   = models.DateField(auto_now_add=True)
+
+    class Meta:
+        db_table = 'users'
+
+class Order(models.Model):
+    user_id            = models.ForeignKey(User, on_delete=models.CASCADE, related_name="User_Order")
+    products_models_id = models.ForeignKey("products.ProductModel", on_delete=models.CASCADE, related_name="ProductModel_Order")
```

Comment:
1. model에서 foreignkey를 만들 때에는 _id를 빼고 만들어 주세요.
2. 필드명과 related_name을 수정해서 제안 드렸습니다. 확인해 해주세요.

```suggestion
    user            = models.ForeignKey(User, on_delete=models.CASCADE, related_name="orders")
    product_model = models.ForeignKey("products.ProductModel", on_delete=models.CASCADE, related_name="orders")
```

Reply:
수정했습니다!